web-react-ts/src/pages/services/banking/anagkazo/ConfirmAnagkazoBanking.tsx

```diff
@@ -23,13 +23,9 @@ type FormOptions = {
 }
 
 type Defaulter = {
-  bacenta: {}
-  id: string
   leader: { pictureUrl: string; fullName: string }
-  meetingDay: { day: string }
   name: string
   services: [{}]
-  __typename: string
```

Comment:
type of typename should be ChurchLevel

Reply:
Type of Bacenta should also be something else. NOt this empty object.

Reply:
I have removed that code